app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt

```diff
@@ -163,16 +196,19 @@ class HomeFragment :
     }
 
     private fun FragmentHomeBinding.setupView() {
-        layoutManager = HomeScreenLayoutManager(requireContext())
-
-        recyclerViewHome.adapter = contentAdapter
-        recyclerViewHome.doOnAttach {
-            recyclerViewHome.layoutManager = layoutManager
-        }
-        recyclerViewHome.doOnDetach {
-            recyclerViewHome.layoutManager = null
+        recyclerLayoutManager = HomeScreenLayoutManager(requireContext())
+        recyclerViewHome.apply {
+            recyclerViewHome.adapter = contentAdapter
+            doOnAttach {
+                layoutManager = recyclerLayoutManager
+            }
+            doOnDetach {
```

Comment:
can you leave a comment on why we need to do this?

Reply:
as you can see this logic already here
but any way we need this as our fragment reused 
view destroying and creating so we need to attach and detach adapter and manager

Reply:
attachAdapter is not what you need?

Reply:
i will check but here we do `layoutManager = recyclerLayoutManager / null`

Reply:
yes this works thanks!

Reply:
but it will be removed after all =(